boa_engine/fuzz/fuzz_targets/parser-idempotency.rs

```diff
@@ -0,0 +1,59 @@
+#![no_main]
+
+mod common;
+
+use crate::common::FuzzData;
+use boa_engine::syntax::Parser;
+use boa_interner::ToInternedString;
+use libfuzzer_sys::fuzz_target;
+use std::error::Error;
+use std::io::Cursor;
+
+fn do_fuzz(mut data: FuzzData) -> Result<(), Box<dyn Error>> {
```

Comment:
Even if the usage of the function could be considered straightforward, adding some documentation could be useful. A link to the libfuzzer documentation or to the cargo-fuzz documentation where this usage is explained would be nice.